edx_sga/sga.py

```diff
@@ -174,7 +176,8 @@ def student_view(self, context=None):
         """
         context = {
             "student_state": json.dumps(self.student_state()),
-            "id": self.location.name.replace('.', '_')
+            "id": self.location.name.replace('.', '_'),
+            "max_file_size": getattr(settings, "STUDENT_FILEUPLOAD_MAX_SIZE", self.STUDENT_FILEUPLOAD_MAX_SIZE)
```

Comment:
Yeah, they have it overridden in their pylint rc file: https://github.com/edx/edx-platform/blob/master/pylintrc#L78, but we still prefer to stick with vanilla pep8 when we can.